app/__init__.py

```diff
@@ -146,13 +147,41 @@ def auth():
         # auth processors always assume a valid Authorization in header, if any
         headers = auth.process(request, headers)
     except jwt.ExpiredSignatureError:
-        current_app.logger.warning("Error while authenticating request", exc_info=True)
-        return Response(json.dumps({"error": "token_expired"}), status=401)
+        current_app.logger.warning(
+            f"Error while authenticating request, token expired. Target: {auth_arg}",
+            exc_info=True,
+        )
+        message = {
+            "error": "authentication",
+            "message": "token expired",
+            "target": auth_arg,
+        }
+        return Response(json.dumps(message), status=401)
+    except AttributeError as error:
+        if "access_token" in str(error):
+            current_app.logger.warning(
+                (
+                    "Error while authenticating request, can't "
+                    f"refresh access token. Target: {auth_arg}"
+                ),
+                exc_info=True,
+            )
+            message = {
+                "error": "authentication",
+                "payload": "can't refresh access token",
```

Comment:
```suggestion
                "message": "can't refresh access token",
```
to be consistent with the above